components/src/core/components/Calendar/Calendar.vue

```diff
@@ -32,7 +32,7 @@ export default defineComponent({
     modelValue: {
       type: Object as PropType<Date>,
       default: () => {
-        return freshDate();
+        return undefined;
```

Comment:
Default to undefined to distinguish between "no value provided" vs "explicit current date provided". This allows min/max constraints to apply when no `modelValue` is given, while still prioritizing explicit `modelValue` (including current date) over constraints.

The `modelValue` is only used to determine the `selectedDate`, so the default value is handled within that logic.